src/wallet/rpc/spend.cpp

```diff
@@ -1302,7 +1302,7 @@ RPCHelpMan send()
                     ParseOutputs(outputs),
                     InterpretSubtractFeeFromOutputInstructions(options["subtract_fee_from_outputs"], outputs.getKeys())
             );
-            CMutableTransaction rawTx = ConstructTransaction(options["inputs"], request.params[0], options["locktime"], rbf);
+            CMutableTransaction rawTx = ConstructTransaction(options["inputs"], request.params[0], options["locktime"], rbf, DEFAULT_RAWTX_VERSION);
```

Comment:
I think perhaps the wallet's default tx version should be specified in a separate constexpr that lives in the wallet code?

Reply:
Done